packages/core/src/lib/TwaManifest.ts

```diff
@@ -282,6 +285,7 @@ export class TwaManifest {
       enableNotifications: DEFAULT_ENABLE_NOTIFICATIONS,
       shortcuts: shortcuts,
       webManifestUrl: webManifestUrl.toString(),
+      features: {locationDelegation: DEFAULT_ENABLE_LOCATIONS ? {} : undefined},
```

Comment:
I'm not quite sure what you're doing here. How come we don't need to do the same for the appsFlyer feature?